modules/base/errors/module.py

```diff
@@ -49,308 +51,662 @@ async def on_command_error(
+                return (
+                    _(ctx, "Bad argument"),
+                    _(ctx, "PartialEmoji conversion failure"),
+                    True,
+                )
+            # Exception raised when a boolean argument was not convertable.
+            elif isinstance(error, commands.BadBoolArgument):
+                return (
+                    _(ctx, "Bad argument"),
+                    _(ctx, "Bad bool argument"),
+                    True,
+                )
+            # Just in case we missed something
+            else:
+                return (
+                    _(ctx, "User input error"),
+                    _(ctx, "Bad argument"),
+                    True,
+                )
+        # Exception raised when a typing.Union converter fails for all its associated types.
+        elif isinstance(error, commands.BadUnionArgument):
+            return (
+                _(ctx, "User input error"),
+                _(ctx, "Bad Union argument"),
                 True,
             )
+        # Exception raised when a parsing or conversion failure is encountered on an argument to pass into a command.
+        elif isinstance(error, commands.ArgumentParsingError):
+            # An exception raised when the parser encounters a quote mark inside a non-quoted string
+            if isinstance(error, commands.UnexpectedQuoteError):
+                return (
+                    _(ctx, "Argument parsing error"),
+                    _(ctx, "Unexpected quote error"),
+                    True,
+                )
+            # An exception raised when a space is expected after the closing quote in a string but a different character is found.
+            elif isinstance(error, commands.InvalidEndOfQuotedStringError):
+                return (
+                    _(ctx, "Argument parsing error"),
+                    _(ctx, "Invalid end of quoted string error"),
+                    True,
+                )
+            # An exception raised when a quote character is expected but not found.
+            elif isinstance(error, commands.ExpectedClosingQuoteError):
+                return (
+                    _(ctx, "Argument parsing error"),
+                    _(ctx, "Unexpected quote error"),
+                    True,
+                )
+            # Just in case we missed something
+            else:
+                return (
+                    _(ctx, "User input error"),
+                    _(ctx, "Argument parsing error"),
+                    False,
+                )
+        # Just in case we missed something
+        else:
+            return (
+                _(ctx, "Nextcord library error"),
+                _(ctx, "User input error"),
+                False,
+            )
 
-        # other exceptions
-        return (
-            type(error).__name__,
-            _(ctx, "An unexpected error occurred"),
-            True,
+    async def handle_log(
```

Comment:
This should have `@staticmethod` decorator

Reply:
fixed in the force push